workspaces/adventure-pack/goodies/java/src/digits_int/AP.java

```diff
@@ -0,0 +1,45 @@
+package digits_int;
+
+import iterable_int_stream.IterableIntStream;
+import java.util.stream.IntStream;
+
+public final class AP {
+
+  /**
+   * Get the digits in the given integer. The digits are returned as an
+   * {@link IterableIntStream integer stream} with the least significant digit as the first element
+   * in the stream. For example, for integer 27363, [3,6,3,7,2] is returned.
+   * @param number Integer of interest.
+   * @return The {@link IterableIntStream stream} of digits.
+   * @throws Exception In case of negative integers.
+   */
+  public static IterableIntStream digits(int number) throws Exception {
```

Comment:
We shouldn't declare the function as throwing `Exception`, isn't that a checked exception? I believe this will require any code that uses this function to also declare itself as throwing `Exception`.

```suggestion
  public static IterableIntStream digits(int number) {
```

Reply:
Addressed.